Assets/HotmaxEditorVR/builder/prop/rotationControl.cs

```diff
@@ -155,60 +105,112 @@ void updateRotationModeEvent(stateManager.rotationModes value)
     void triggerUnclicked(object sender, ClickedEventArgs e)
     {
         init._stateManagerMutatorRef.SET_ROTATION_GIZMO_IS_SELECTED(false);
```

Comment:
reference stateManagerMutatorRef at top of class.. i idon't think having a bunch of init.<whatever> sprinkled around the body of classes is a good idea longterm.